src/targets/chrome/get-selector-box-size.js

```diff
@@ -12,6 +12,27 @@ const getSelectorBoxSize = (window, selector) => {
     return !isWrapper;
   };
 
+  const isVisisble = element => {
+    let style = {};
+    try {
+      style = window.getComputedStyle(element);
+    } catch (error) {
+      return false;
+    }
+
+    if (
```

Comment:
You can just return this if statement instead

Reply:
I don't get it 😕

Reply:
More like 
```js
return style.visibility === 'hidden' ||
      style.display === 'none' ||
      style.opacity === '0' ||
      style.width === '0px' ||
      style.height === '0px';
```

Reply:
It would need a negation, something like this:

```js
return !(expression);
```

Reply:
Done! ✅